src/pages/registration/resource_type_tab/SelectRegistrationTypeField.tsx

```diff
@@ -220,79 +228,156 @@ export const SelectRegistrationTypeField = () => {
             </IconButton>
           )}
         </Box>
-        <Box sx={{ display: 'flex', gap: '0.5rem', alignItems: 'center', justifyContent: 'end', mb: '1rem' }}>
-          <FilterVintageIcon
-            color="primary"
-            titleAccess={t('registration.resource_type.nvi.can_give_publication_points')}
-            fontSize="small"
+        <Box
+          sx={{
+            display: 'flex',
+            justifyContent: 'space-between',
+            flexWrap: 'wrap',
+            padding: '0.5rem',
+            gap: '0.5rem',
+            my: '1rem',
```

Comment:
Ble kanskje vel mye tomrom?
```suggestion
            gap: '0.5rem',
            mb: '1rem',
```

Reply:
Det ble det. Fikk ikke med meg at det ikke var PAper helt i starten, så la på padding igjen :|